README.md

```diff
@@ -116,7 +116,7 @@ In addition, you can find:
 ## Contributing
 
 This project welcomes contributions and suggestions. Most contributions require you to agree to a
-Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
+Contributor License Agreement (CLA) declaring that you have the right to, and do, grant us
```

Comment:
Please do not modify this section. The changes in other sections are good.

Reply:
Sure i will revert the changes